template-for-repository/proofs/cbmc_ensure__signed_overflow_check/cbmc_ensure__signed_overflow_check_harness.c

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+/**
+ * A negative test for --signed-overflow-check flag
+ */
+void cbmc_ensure__signed_overflow_check_harness() {
+    int overflow, offset;
+    overflow += offset;
+    overflow -= offset;
```

Comment:
We should split into two harnesses.

Reply:
I made this change pushed several underflow harnesses (for other overflow flags as well), but now there is no 1:1 match between flags and negative tests, so disabling specific negative tests might be a little more complicated. Let's discuss this.